docs/pages/management/security/proxy-protocol.mdx

```diff
@@ -1,88 +1,134 @@
 ---
-title: PROXY protocol
-description: How to securely configure PROXY protocol usage with Teleport
+title: Run Teleport Behind a Layer 4 Load Balancer
```

Comment:
Feels like the title should mention proxy protocol, no?

Reply:
Agree that it makes more sense for this guide to focus on proxy protocol and not L4 load balancing in general. There are other ways to achieve source IP preservation with L4 load balancers (e.g., AWS NLBs are L4 load balancers that can do source IP preservation with SNAT).